### ISSUE TITLE
web_access.log is created with mode 666 in development

### ISSUE THREAD

**Issue by Hobbsee** [flapjack/flapjack#809]
Need to check if this also happens on production:

```
log ➤ rm *.log
log ➤ bundle exec ../bin/flapjack --config ../etc/flapjack_config.toml server start
log ➤ ls -lh
total 16
-rw-r--r--  1 sarah  staff   1.3K  6 Mar 13:55 flapjack.log
drwxr-xr-x  3 sarah  staff   102B  6 Mar 13:55 run
-rw-rw-rw-  1 sarah  staff    66B  6 Mar 13:55 web_access.log
```

**Comment by ali-graham** [comment]
The umask stuff probably hasn't been ported forward from the maint branch.

**Comment by Hobbsee** [comment]
Fixed, cherry picked 80aa33d6
